packages/payment-detection/src/any/any-to-any-proxy-info-retriever.ts

```diff
@@ -179,4 +155,17 @@ export default class ProxyERC20InfoRetriever
 
     return Promise.all(eventPromises);
   }
+
+  protected getFeeFilter(): ethers.providers.Filter {
```

Comment:
Signature of `TransferWithReferenceAndFee` is different between `eth-fee-proxy` and `erc20-fee-proxy` so we need a different way to filter them.